### PR TITLE
Fix jared

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
👍 

But we need to understand why these tests failed *Asap*.

File: build/Targets/Settings.props [hunk 1]
```diff
@@ -46,6 +46,14 @@
     <!-- To copy .dll.config and .exe.config files of the referenced projects to the output dir 
          (see https://github.com/Microsoft/msbuild/issues/1307) -->
     <AllowedReferenceRelatedFileExtensions>$(AllowedReferenceRelatedFileExtensions);.dll.config;.exe.config</AllowedReferenceRelatedFileExtensions>
+
+    <!--
```
Comment (jasonmalinowski) [inline build/Targets/Settings.props]: Looks like this is unrelated from your machine?

File: src/EditorFeatures/VisualBasicTest/ExtractMethod/ExtractMethodTests.ControlFlowAnalysis.vb [hunk 2]
```diff
@@ -282,7 +282,7 @@ End Class</text>
             End Function
 
             <WorkItem(540154, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/540154")>
-            <Fact, Trait(Traits.Feature, Traits.Features.ExtractMethod)>
+            <Fact(Skip:= "https://github.com/dotnet/roslyn/issues/15223"), Trait(Traits.Feature, Traits.Features.ExtractMethod)>
```
Comment (genlu) [inline src/EditorFeatures/VisualBasicTest/ExtractMethod/ExtractMethodTests.ControlFlowAnalysis.vb]: Based on the test result and issue #15223, it should be `Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.ExtractMethod.ExtractMethodTests+FlowAnalysis.BugFix6313_6` instead.
